gatsby/content/blog/2022/11/2022-11-22-synapse-1.72.mdx

```diff
@@ -0,0 +1,70 @@
+It seems like this blog didn't self update through AI for the 2 precedent updates so this post will cover 1.70 to 1.72, sorry if this is a bit long but it's been a while :)
+
+## Announcements
+
+### Dropping support for PostgreSQL 10
+
+PostgreSQL 10 being End Of Life, Synapse will not support it beginning with version 1.72 so please upgrade your DB if not already done.
+
+### Legacy Prometheus metric names disabled and to be removed
+
+In previous versions of Synapse, some Prometheus metrics are emitted under two different names, with one of the names being older but non-compliant with OpenMetrics and Prometheus conventions and one of the names being newer but compliant.
+
+Synapse v1.71 have the old metric names switched off by default.
+
+For now it's still possible to get them by using `enable_legacy_metrics: true`, however if you still use them you should really migrate because it will be removed in v1.73.
+
+You can find the full list of renamed metrics [here](https://matrix-org.github.io/synapse/v1.69/metrics-howto.html#renaming-of-metrics--deprecation-of-old-names-in-12).
+
+### Changes to the events received by application services (interest)
+
+[MSC3905](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/3905-appservice-only-interested-in-local-users.md) has been implemented, Synapse now only considers local users to be interesting. In other words, the users namespace regex is only be applied against local users of the homeserver.
+
+Please note, this probably doesn't affect the expected behavior of your application service, since an interesting local user in a room still means all messages in the room (from local or remote users) will still be considered interesting.
+
+You can find a bit more info in the MSC and in the (upgrade notes)[https://matrix-org.github.io/synapse/v1.71/upgrade.html#changes-to-the-events-received-by-application-services-interest].
+
+## The new stuffs
+
+### Threads, threads, threads!
+
+Several MSCs related to threads got implemented:
+- [MSC3856](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/3856-threads-list-api.md) providing an API to fetch threads and related metadata.
+- [MSC3771](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/3771-read-receipts-for-threads.md) and [MSC3773](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/3773-notifications-for-threads.md) implementing per thread read receipts and per thread notifications count.
+- [MSC3874](https://github.com/matrix-org/matrix-spec-proposals/blob/justjanne/messages-endpoint-threads-filter/proposals/3874-messages-endpoint-threads-filter.md) allows to filter out messages belonging to threads from the main timeline. Along with MSC3856, this should noticeably improve performance of rooms that use threads heavily.
+
+This should significantly improve user experience related to threads, being through behavior or performance impact.
+
+### Bounding events together
```

Comment:
I don't think "Bounding" is the correct word here, but I'm not 100% sure what is -- maybe "Bundling"?

Reply:
I am not sure TBH, any other feedback from people regarding that ?

Reply:
Replacing this with "Linking" which sounds more fitting to me